### ISSUE TITLE
No Focus Visible - LLM Controls

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#3476]
**Issue Description**: There is no visible focus outline on the buttons to adjust the LLM controls in the chat window.

**WCAG Criteria:** [2.4.7 Focus Visible - AA](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible)

**Notes:** Visible focus outline assists keyboard and other non-mouse users in navigating the interface effectively.
<hr/>
<img width="607" alt="Screenshot 2024-07-29 at 4 20 41 PM" src="https://github.com/user-attachments/assets/5e670712-fe33-421d-b5bb-944d8cfc29e2">

<img width="1514" alt="Screenshot 2024-07-29 at 4 22 58 PM" src="https://github.com/user-attachments/assets/bd08722e-0700-4a3e-ad9a-97d2a8622fe1">

**Comment by danny-avila** [comment]
https://github.com/user-attachments/assets/b31ee5b8-6907-4fdc-b5d9-8a2a3b02d24a

closed by #3536 

Styling for this will be improved by https://github.com/danny-avila/LibreChat/pull/3434 when merged
